models/marts/core/dim_customers.sql

```diff
@@ -0,0 +1,47 @@
+with customers as (
+    select * from {{ ref('stg_customers') }}
+),
+
+orders as (

```

Comment:
you want to see if you can incorporate everything you need about orders into your `fct_orders` model and build on that 👍🏻